tests/istio.mk

```diff
@@ -227,10 +227,10 @@ test/local/auth/e2e_split_horizon: out_dir generate_e2e_yaml
 		--cluster_wide ${E2E_ARGS} ${T} ${EXTRA_E2E_ARGS} ${CAPTURE_LOG}
 
 test/local/noauth/e2e_mixer_envoyv2: export EXTRA_HELM_SETTINGS=--set mixer.adapters.stdio.enabled=false
-test/local/noauth/e2e_mixer_envoyv2: out_dir generate_e2e_yaml
+test/local/noauth/e2e_mixer_envoyv2: out_dir generate_installer_e2e_yaml
 	set -o pipefail; go test -v -timeout 35m ./tests/e2e/tests/mixer \
 	--auth_enable=false --egress=false --ingress=false --rbac_enable=false \
-	--cluster_wide ${E2E_ARGS} ${T} ${EXTRA_E2E_ARGS} ${CAPTURE_LOG}
+	--installer=istio-installer --use_galley_config_validator=false --cluster_wide ${E2E_ARGS} ${T} ${EXTRA_E2E_ARGS} ${CAPTURE_LOG}
```

Comment:
what's wrong with galley config validator? is there a tracking issue?

Reply:
It has been too long for me to recall exactly, but I believe it was a known issue at the time that has likely been resolved. Since galley isn't being tested here, I'm not sure it matters, but we can remove this in a follow-on.